### PR TITLE
update docs for estimator and new tests

### PR DESCRIPTION
**Pull request by brilee**
@sethtroisi @amj

### CODE REVIEW

**Comment by Kashomon** [comment]
Note: You'll have to re-build and push the docker file V2, or the tests won't use it. 

Also, you can totally get rid of the V1 Dockerfile. It's dead code now.

File: README.md [hunk 1]
```diff
@@ -87,11 +87,11 @@ pip3 install -r requirements.txt
 
 Then, you'll need to choose to install the GPU or CPU tensorflow requirements:
 
-- GPU: `pip3 install "tensorflow-gpu>=1.5,<1.6"`.
+- GPU: `pip3 install "tensorflow-gpu>=1.7,<1.8"`.
   - *Note*: You must install [CUDA
     9.0].(https://developer.nvidia.com/cuda-90-download-archive) for Tensorflow
     1.5.
```
Comment (amj) [inline README.md]: says TF 1.5, should be 1.5+?

File: README.md [hunk 2]
```diff
@@ -251,7 +255,9 @@ This command creates a random model, which appears at .
 
 ```bash
 export MODEL_NAME=000000-bootstrap
-python3 main.py bootstrap --model-save-path="gs://$BUCKET_NAME/models/$MODEL_NAME"
+python3 main.py bootstrap \
+  --working-dir=estimator_working_dir
```
Comment (amj) [inline README.md]: Does this need to exist in advance?  If not, you can mention it's created at runtime

File: README.md (hunk 1 above)
Comment (brilee) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (brilee) [reply README.md]: done

**Pull request merged**